### PR TITLE
SEO-2363-JavaScript-charts-NPM-Moneykeywords

### PR DESCRIPTION
**Pull request by Gowri-V-L**

### CODE REVIEW

File: controls/charts/ReadMe.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 # ej2-charts
 
-The Chart control is used to visualize the data with user interactivity and provides customizing options to configure the data visually. It can bind data from  datasource such as array of JSON objects , `OData web services` or
+The [Chart control](https://www.syncfusion.com/javascript-ui-controls/js-charts) is used to visualize the data with user interactivity and provides customizing options to configure the data visually. It can bind data from  datasource such as array of JSON objects , `OData web services` or
```
Comment (Girija-Rajendran) [inline controls/charts/ReadMe.md:3]: should add the platform name **JavaScript** Chart

File: controls/charts/ReadMe.md [hunk 2]
```diff
@@ -17,26 +17,22 @@ npm install @syncfusion/ej2-charts
 ## Components included
 
 Following list of components are available in the package
-*	Chart  - Feature-rich chart control with built-in support for over 25 chart types, technical indictors, trendline, zooming, tooltip, selection, crosshair and trackball.
```
Comment (Girija-Rajendran) [inline controls/charts/ReadMe.md:20]: remove the extra space and add the platform name before the control name

File: controls/charts/ReadMe.md [hunk 3]
```diff
@@ -100,11 +96,11 @@ Following list of components are available in the package
    * [**Tooltip and Trackline**](https://ej2.syncfusion.com/demos/?utm_source=npm&utm_campaign=chart#/material/sparkline/default.html) - Supports interactive features such as tooltip and trackline.
 
 ### Smith chart
-   * [**Series**](https://ej2.syncfusion.com/demos/?utm_source=npm&utm_campaign=chart#/material/smithchart/default.html) - Supports any number of series to display data points.
```
Comment (Girija-Rajendran) [inline controls/charts/ReadMe.md:103]: why did you remove the demo link reference?

File: controls/charts/ReadMe.md [hunk 4]
```diff
@@ -122,4 +118,30 @@ Check the license detail [here](https://github.com/syncfusion/ej2/blob/master/li
...
+
+## About Syncfusion
+
+Founded in 2001 and headquartered in Research Triangle Park, N.C., Syncfusion has more than 28000+ companies and more than 1 million users, including large financial institutions, Fortune 500 companies, and global IT consultancies.
```
Comment (Girija-Rajendran) [inline controls/charts/ReadMe.md:137]: remove + ........**more than 28000 companies**.....

File: controls/charts/ReadMe.md (hunk 2 above)
Comment (Gowri-V-L) [reply controls/charts/ReadMe.md:20]: modified

File: controls/charts/ReadMe.md (hunk 3 above)
Comment (Gowri-V-L) [reply controls/charts/ReadMe.md:103]: because FT link replaced here and demo taken into bottom part

Comment (Gowri-V-L) [reply controls/charts/ReadMe.md:103]: replaced with UG so, and now demo added

File: controls/charts/ReadMe.md (hunk 4 above)
Comment (Gowri-V-L) [reply controls/charts/ReadMe.md:137]: modified
